common/lib/capa/capa/capa_problem.py

```diff
@@ -187,32 +187,32 @@ def __init__(self, problem_text, id, capa_system, capa_module,  # pylint: disabl
 
         # construct script processor context (eg for customresponse problems)
         if minimal_init:
-            self.context = {'script_code': ""}
```

Comment:
It turns out we don't need it anymore since I am now excluding more initialization code.